frontend/templates/troubleshooting/prerendered.tsx

```diff
@@ -0,0 +1,53 @@
+import { Box, chakra, SystemStyleObject } from '@chakra-ui/react';
+
+const Prerendered = chakra(function Prerendered({
+   html,
+   className,
+}: {
+   html: string;
+   className?: string;
+}) {
+   const renderStyles: SystemStyleObject = {
+      '.headerContainer': {
+         display: 'flex',
+         alignItems: 'baseline',
+         marginBottom: 2,
+      },
+
+      '.selfLink': {
+         display: 'none',
+      },
+
+      h3: {
+         fontSize: 'xl',
+         lineHeight: '1.2',
+      },
+
+      'h3,h4': {
+         fontWeight: 590,
+      },
+
+      'h4,h5': {
+         fontSize: 'md',
+         lineHeight: '1.25',
+      },
+
+      p: {
+         lineHeight: '1.38',
+         fontWeight: 'regular',
+         fontSize: '16px',
+         color: 'gray.700',
+         alignSelf: 'stretch',
+      },
+   };
```

Comment:
Minor: you can pull this out to the top level

Reply:
[d3af21e](https://github.com/iFixit/react-commerce/pull/1305/commits/d3af21ecd54c849054a6652f709273cd1510cf6a)